tester/paddle_to_torch/mapping.json

```
@@ -617,6 +617,21 @@
             "y": "other"
         }
     },
+    "paddle.multinomial": {
+        "torch_api": "torch.multinomial",
+        "set_defaults": {
+            "num_samples": 1
+        },
+        "paddle_torch_args_map": {
+            "x": "input",
+            "num_samples": "num_samples",
+            "replacement": "replacement"
+        },
+        "torch_args": [],
+        "torch_kwargs": {
+            "generator": "None"
+        }
```

torch 的 generator 默认为 None，可不必设置此字段：
```suggestion
        }
```

---

Done